packages/visual-editor/src/docs/components.md

```diff
@@ -0,0 +1,623 @@
+---
+title: Pre-Built Components
+outline: 2
+---
+
+## BannerSection
+
+The Banner Section component displays a single, translatable line of rich text. It's designed to be used as a simple, full-width banner on a page. Avaliable on Location templates.
+
+![Preview of the BannerSection component](../components/testing/screenshots/BannerSection/%5Bdesktop%5D%20default%20props%20with%20empty%20document.png)
```

Comment:
Idk if there's anything you can do, but the screenshot is a full page so there's a bunch of whitespace, rather than just showing the component. Maybe there's a way our screenshot stuff can be adjusted later. Nothing to do here, just pointing out.